sdcm/sct_config.py

```diff
@@ -168,6 +168,12 @@ class SCTConfiguration(dict):
                      Automatically lookup AMIs and repo links for formal versions.
                      WARNING: can't be used together with 'scylla_repo' or 'ami_id_db_scylla'"""),
 
+        dict(name="oracle_scylla_version", env="SCT_ORACLE_SCYLLA_VERSION",
```

Comment:
please recreate the docs:
`SCT_CLUSTER_BACKEND=aws hydra conf-docs -o markdown > docs/configuration_options.md`

Reply:
Done